pkg/console/validator.go

```diff
@@ -274,6 +276,27 @@ func checkToken(token string) error {
 	return nil
 }
 
+func checkSystemSettings(systemSettings map[string]string) error {
+	if systemSettings == nil {
+		return nil
+	}
+
+	allowList := config.GetSystemSettingsAllowList()
+	for systemSetting, _ := range systemSettings {
+		isValid := false
+		for _, allowSystemSetting := range allowList {
```

Comment:
NIT: Can use a map to avoid looping `allowList` for each setting.

```
func GetAllowedSystemSettings() map[string]interface{} {
	return map[string]interface{}{
		"additional-ca":             nil,
        ...
    }
}

func checkSystemSettings(systemSettings map[string]string) error {
	if systemSettings == nil {
		return nil
	}

	allowedSystemSettings := GetAllowedSystemSettings()
	for systemSetting, _ := range systemSettings {
		if _, ok := allowedSystemSettings[systemSetting]; !ok {
			return errors.Errorf(ErrMsgSystemSettingsUnknown, systemSetting)
		}
	}
	return nil
}
```

Reply:
I found this benchmark
`map` take lots of time of it.
so i chose linear search and keep readability
https://stackoverflow.com/questions/15323767/does-go-have-if-x-in-construct-similar-to-python